source/configuration_manual/acl.rst

```diff
@@ -251,24 +251,26 @@ Prevent all users from deleting their Spam folder (notice no x flag)
 
   INBOX.Spam owner lrwstipeka
 
-List cache
-^^^^^^^^^^
+List Cache
+==========
 
 ``dovecot-acl-list`` file lists all mailboxes that have ``l`` rights assigned.
 If you manually add/edit ``dovecot-acl`` files, you may need to delete the
 ``dovecot-acl-list`` to get the mailboxes visible.
 
 Dictionaries
-^^^^^^^^^^^^
-
-In order for an ACL to be fully useful, it has to be communicated to IMAP clients.
-For example, if you use ACL to share a mailbox to another user, the client has to
-be explicitly told to check out the other user's mailbox too, as that one is shared.
-Placing the ACL file makes the ACL effective, but ``dovecot`` doesn't take care of
-the user->shared mailboxes mapping out of the box, and as a result, it won't publish
-shared mailboxes to clients if this is not set up.
-You have to configure this manually by defining an appropriate
-:ref:`dictionary <dict>` to store the share map.
-
-Certain dictionary backends are writable by ``dovecot``, so when you establish
+============
+
+In order for an ACL to be fully useful, it has to be communicated to IMAP
+clients. For example, if you use ACL to share a mailbox to another user, the
+client has to be explicitly told to check out the other user's mailbox too, as
+that one is shared.
+
+Placing the ACL file makes the ACL effective, but Dovecot doesn't take care of
+the user to shared mailboxes mapping out of the box, and as a result, it won't
+publish shared mailboxes to clients if this is not set up. You have to
+configure this manually by defining an appropriate :ref:`dictionary <dict>` to
+store the share map.
+
+Certain dictionary backends are writable by Dovecot, so when you establish
 an ACL using ``doveadm``, a dictionary entry is added along to the ACL.
```

Comment:
This sentence doesn't really make sense. Maybe just remove it entirely?

Reply:
Removed.